<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #99EDC3</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>811</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 255, 57);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>809</width>
      <height>549</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="ADDTOCART">
      <property name="styleSheet">
       <string notr="true">font: 87 12pt &quot;Montserrat Black&quot;;
background-color: #120A8F;
color:white;</string>
      </property>
      <property name="text">
       <string>add to cart</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 87 12pt &quot;Montserrat Black&quot;;
color:white;</string>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
